storybook/cypress/e2e/modal.cy.ts

```diff
@@ -29,6 +28,16 @@ describe("Modal", () => {
     cy.wait(200).matchImageSnapshot();
   });
 
+  it.only("should have the propper role set by modal component", () => {
+    cy.get("dso-modal")
+      .invoke("attr", "component-role", "alert")
+      .shadow()
+      .find(".dso-modal")
+      .should("have.attr", "role", "alert")
+      .and("have.attr", "aria-modal", "true")
+      .and("have.attr", "aria-labelledby");
```

Comment:
Deze 2 mogen weg want die worden al in de `should be accessible` test gecheckt, dus die voegen hier niets toe.

Reply:
Ik zie de aanpassing niet.

Reply:
Nu wel?